shanoir-ng-import/src/main/java/org/shanoir/ng/importer/ImporterManagerService.java

```diff
@@ -129,39 +129,45 @@ public void manageImportJob(final Long userId, final HttpHeaders keycloakHeaders
 			if (importJob.isFromPacs()) {
 				importJobDir = createImportJobDir(userImportDir.getAbsolutePath());
 				// at first all dicom files arrive normally in /tmp/shanoir-dcmrcv (see config DicomStoreSCPServer)
-				downloadAndMoveDicomFilesToImportJobDir(importJobDir, patients);
+				downloadAndMoveDicomFilesToImportJobDir(importJobDir, patients, event);
 				// convert instances to images, as already done after zip file upload
-				imagesCreatorAndDicomFileAnalyzer.createImagesAndAnalyzeDicomFiles(patients, importJobDir.getAbsolutePath(), true);
+				imagesCreatorAndDicomFileAnalyzer.createImagesAndAnalyzeDicomFiles(patients, importJobDir.getAbsolutePath(), true, event);
 			} else if (importJob.isFromShanoirUploader()) {
 				importJobDir = new File(importJob.getWorkFolder());
 				// convert instances to images, as already done after zip file upload
-				imagesCreatorAndDicomFileAnalyzer.createImagesAndAnalyzeDicomFiles(patients, importJobDir.getAbsolutePath(), false);
+				imagesCreatorAndDicomFileAnalyzer.createImagesAndAnalyzeDicomFiles(patients, importJobDir.getAbsolutePath(), false, event);
 			} else if (importJob.isFromDicomZip()) {
 				// images creation and analyze of dicom files has been done after upload already
 				importJobDir = new File(importJob.getWorkFolder());
 			} else {
 				throw new ShanoirException("Unsupported type of import.");
 			}
 						
-			event.setMessage("Anonymizing dicom..");
+			event.setProgress(0.25F);
 			eventService.publishEvent(event);
+
 			for (Iterator<Patient> patientsIt = patients.iterator(); patientsIt.hasNext();) {
 				Patient patient = patientsIt.next();
 				// perform anonymization only in case of profile explicitly set
 				if (importJob.getAnonymisationProfileToUse() != null && !importJob.getAnonymisationProfileToUse().isEmpty()) {
 					ArrayList<File> dicomFiles = getDicomFilesForPatient(importJob, patient, importJobDir.getAbsolutePath());
 					final Subject subject = patient.getSubject();
-					if (subject != null) {
-						final String subjectName = subject.getName();
-						try {
-							ANONYMIZER.anonymizeForShanoir(dicomFiles, importJob.getAnonymisationProfileToUse(), subjectName, subjectName);
-						} catch (Exception e) {
-							LOG.error(e.getMessage(), e);
-							throw new ShanoirException("Error during anonymization.");
-						}						
-					} else {
+
+					if (subject == null) {
 						LOG.error("Error: subject == null in importJob.");
-						throw new ShanoirException("Error: subject == null in importJob.");						
+						throw new ShanoirException("Error: subject == null in importJob.");
+					}
+
+					final String subjectName = subject.getName();
+
+					event.setMessage("Anonymizing DICOM files for subject [" + subjectName + "]...");
```

Comment:
Please change here to Pseudonymizing DICOM files, same above, today we should not use the word "anonymization" in our context please (legacy bug)